public/r2/v1/swagger/swagger.json

```diff
@@ -198,28 +198,29 @@
                     "application/json"
```

Comment:
There were all kinds of swagger compilation problems with this file so I fixed them.